ecs-cli/modules/utils/compose/reconcile_container_def.go

```diff
@@ -93,6 +92,7 @@ func reconcileContainerDef(inputCfg *adapter.ContainerConfig, ecsConDef *Contain
 	memLimit := inputCfg.Memory
 	memRes := inputCfg.MemoryReservation
 	healthCheck := inputCfg.HealthCheck
+	var resourceRequirements []*ecs.ResourceRequirement
```

Comment:
Question: what if we supported resource type in ECS Params instead of having a specific GPU value?

Reply:
talked offline, keeping it as is!